addons/calendar/calendar.py

```diff
@@ -1440,7 +1441,13 @@ def get_interval(self, cr, uid, ids, date, interval, tz=None, context=None):
         elif interval == 'time':
             dummy, format_time = self.get_date_formats(cr, uid, context=context)
             res = date.strftime(format_time + " %Z")
-        return res
+        return res.decode(getpreferredencoding(do_setlocale=False))
```

Comment:
Ok, thanks for `tools.ustr()`, I didn't know this method.

I will try to fix that with `babel`. As these strings are meant to be used in email templates, if I use the context language to localize them, will it be ok if the templates are translated? I wouldn't want to introduce a language mismatch